docs/book/src/view/04_iteration.md

```diff
@@ -26,7 +26,7 @@ view! { cx,
     <ul>
         {values.into_iter()
             .map(|n| view! { cx, <li>{n}</li>})
-            .collect::<Vec<_>>()}
+            .collect_view(cx)}
```

Comment:
In this specific case, since the context is talking about collect into a `Vec<_>`, it might be better to let it stand, and add a paragraph like

> Leptos also provides a `.collect_view(cx)` helper function that allows you to collect any iterator of views into `Vec<View>`.

And then include the modified example.